src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+micronaut:
```

Comment:
The problem here is that we have a single config file for all environments. This may become cumbersome for most microservices.

Is there a way to have separate config files and still load them using Micronaut environment magic?